org.lflang/src/org/lflang/TargetProperty.java

```diff
@@ -823,14 +823,14 @@ public static void produceError(String name, String description,
             v.getKeyvalue() == null && v.getArray() == null
                 && v.getLiteral() == null && v.getId() == null
                 && (v.getTime() == 0 || v.getUnit() != TimeUnit.NONE)),
-        STRING("a string", v -> v.getLiteral() != null || v.getId() != null),
+        STRING("a string", v -> v.getLiteral() != null && !isCharLiteral(v.getLiteral()) || v.getId() != null),
```

Comment:
Why not allow chars here? They can be losslessly converted to strings...

Reply:
They can also losslessly be converted to integers, which is I believe less surprising when you come from a C-like language... 

I think we should keep types clearly separated, esp. when allowing chars here doesn't net us anything. If you allow a character literal to be interpreted as a string, then you're opening the way to some more lexer errors, as someone who wants to modify the literal might forget to change the delimiters. I think it also might be easier to teach "LF doesn't support single quoted literals" if we don't muddy the distinction between char literals and strings.

Reply:
OK, I think you're right.